src/scss/includes/search_form.scss

```diff
@@ -227,162 +233,18 @@ input[type="search"] {
   padding: 9px 18px;
 }
 
-.top_bar--small {
-
-  background-size: 0;
-
-  .search_form {
-
-    max-width: 68px;
-
-    .search_form__input {
-      visibility: hidden;
-      padding: 0;
-    }
-
-    .search_form__action {
-      display: none;
-    }
-
-    .search_form__clear {
-      display: none;
-    }
-
-    .search_form__logo {
-      left: 0;
-      top: 0;
-      width: 40px;
-      height: 40px;
-      background-size: 38px;
-      background-position: center center;
-      padding: 33px;
-    }
-  }
-
-  .search_form::before {
-      right: 15px;
-  }
-}
-
 @media (max-width: 640px) {
-  .search_form:before {
-    right: 5px;
-  }
-
-  .search_form {
-    height: 48px;
-    width: calc(100vw - 72px);
-    left: 8px;
-    top:16px;
-    margin: 0;
-
-    .no-burger & {
-      width: calc(100% - 16px);
-    }
-  }
-
-  .search_form__input {
-    font-size: 16px;
-    position: absolute;
-    width: 100%;
-    height: 48px;
-    padding: 0 68px 0 50px;
-    border-radius: 24px;
-    margin-left: 0;
-  }
-
-  .search_form__action {
-    height: 48px;
-    right: 12px;
-    top: 0;
-  }
-
-  .search_form__clear {
-    right: 45px;
-    height: 48px;
-    top: 0;
-    font-size: 14px;
-  }
-
   .search_form__input:invalid {
     padding-right: 8px;
   }
 
-  .autocomplete_suggestions {
-    position: fixed;
-    top: 0;
-    bottom: 0;
-    max-height: none;
-    margin-top: -1px;
-    overflow: auto;
+  .top_bar .autocomplete_suggestions {
+    height: 100vh;
   }
 
   .search_form__logo {
     width: 34px;
     height: 34px;
-    top: 7px;
-    left: 8px;
-    background-size: 34px 34px;
-  }
-
-  .top_bar--search_focus {
-
-    .search_form:before {
-      display: none;
-    }
-
-    background: url("../images/panel_top.svg");
-    .search_form {
-      width: 100%;
-      top: 4px;
-      left: 0;
-      border-radius: 0;
-      height: 60px;
-    }
-
-    .search_form__input {
-      height: 60px;
-      padding: 0 70px 0 58px;
-      margin: 0;
-    }
-
-    .search_form__input:invalid {
-      padding-right: 8px;
-    }
-
-    .search_form__action {
-      height: 60px;
-      top: 0;
-    }
-
-    .search_form__clear {
-      height: 60px;
-      right: 40px;
-      top: 0;
-    }
-
-    .search_form__logo {
-      display: none;
-    }
-
-    .main_field_return {
-      display: block;
-      left: 10px;
-      top: 20px;
-      font-size: 25px;
-      color: $primary_text;
-      position: fixed;
-    }
-  }
-}
-
-@media (max-width: 380px) {
-  .search_form__input {
-    letter-spacing: -.5px;
-  }
-
-  &::placeholder {
-    font-size: 14px;
   }
```

Comment:
As a result, the placeholder is truncated on some mobiles:
![image](https://user-images.githubusercontent.com/4726554/86481958-47a1ad00-bd51-11ea-987a-48c4ed496f98.png)

(Here on Iphone 6/7/8 simulated by Chromium)

Reply:
What do you think we should do? Having the same font-size for the placeholder and the content is an improvement to me. We can reduce the padding of the magnifiying glass button, but it would reduce its accessibilty. And the problem may be even bigger for other languages, I doubt we can have a solution working for 100% of the cases…
But I agree we should improve this use case which is probably one of the most frequent.

Reply:
If we no longer want to adapt the font-size or letter-spacing, the simplest solution would be to hide the placeholder on small screens.
This is already done for `max-width: 340px` (just a few lines below). We could update this threshold to around 400-450px to be safe.

Technically, it would be acceptable to me. It would make the brand "Qwant Maps" less visible though.

Reply:
I adjusted some paddings to win the missing pixels, I think it's better than deciding to hide the string on so many devices. Here is the result on the IPhone 6/7/8 simulator in Chromium:
![localhost_3000_(iPhone 6_7_8) (4)](https://user-images.githubusercontent.com/243653/86572244-f3761300-bf72-11ea-9568-90645841db6e.png)

And the before/after on my personal phone (360/640):
|Before|After|
|---|---|
|![Screenshot_20200706-101309_Chrome](https://user-images.githubusercontent.com/243653/86572348-1c96a380-bf73-11ea-9f59-4e2216cd4b54.jpg)|![Screenshot_20200706-101023_Chrome](https://user-images.githubusercontent.com/243653/86572374-26200b80-bf73-11ea-873c-b8eb3cf2e0b7.jpg)|